code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -137,6 +137,17 @@
 /obj/item/weapon/reagent_containers/spray/waterflower/attack_self(mob/user) //Don't allow changing how much the flower sprays
 	return
 
+//syndicate water flower
+/obj/item/weapon/reagent_containers/spray/syndiwaterflower

```

Comment:
If you make this a child of the existing water flower, you won't need to define a bunch of things like the icon. It'll also make this fit where the clown's waterflower should!